frontend/store/categories/action.js

```diff
@@ -0,0 +1,27 @@
+import {createApi, fetchBaseQuery} from "@rtk-incubator/rtk-query";
+
+export const categoriesApi = createApi({
```

Comment:
naming and some params will change as soon as I figure out a proper way to separate endpoints, add mutations and add auth token